src/wayland/generated/wayland_wrapper.cpp

```diff
@@ -80,6 +80,10 @@ mw::Callback::Callback(struct wl_resource* resource, Version<1>)
     }
 }
 
+mw::Callback::~Callback()
+{
+}
```

Comment:
Specifically note that the generated `~Callback()` is is empty, where as in the previous PR it tried to clear the resource implementation. This is because the deletion path for objects without requests is different.

Reply:
A shame it's not the more modern C++ `= default`. :)

Reply:
I considered that, but this makes the generator implementation moderately more obvious. Since this whole area is convoluted and needs to get refactored, I decided it was good enough.